deployment/packer/template.js

```diff
@@ -62,6 +62,14 @@
             "instance_type": "t2.large",
             "ssh_username": "ubuntu",
             "ami_name": "mmw-worker-{{timestamp}}-{{user `version`}}",
+            "ami_block_device_mappings": [
+                {
+                    "device_name": "/dev/sdf",
+                    "snapshot_id": "snap-7bf80133",
```

Comment:
I created a newly labeled snapshot based on the previous. Its ID is `snap-3d8f22ab`.